packages/desktop-client/src/components/mobile/transactions/TransactionList.jsx

```diff
@@ -119,6 +145,244 @@ export function TransactionList({
+          onDuplicate={onBatchDuplicate}
+          onLinkSchedule={onLinkSchedule}
+          onUnlinkSchedule={onUnlinkSchedule}
+          onSetTransfer={onSetTransfer}
+          onDelete={onBatchDelete}
+        />
+      )}
     </>
   );
 }
+
+function FloatingActionBar({
+  transactions,
+  selectedTransactions,
+  onClearSelectedTransactions,
+  onEdit,
+  onDuplicate,
+  onLinkSchedule,
+  onUnlinkSchedule,
+  onDelete,
+  style,
+}) {
+  const editMenuTriggerRef = useRef(null);
+  const [isEditMenuOpen, setIsEditMenuOpen] = useState(false);
+  const moreOptionsMenuTriggerRef = useRef(null);
+  const [isMoreOptionsMenuOpen, setIsMoreOptionsMenuOpen] = useState(false);
+  const getMenuItemStyle = item => ({
+    ...styles.mobileMenuItem,
+    color: theme.mobileHeaderText,
+    ...(item.name === 'delete' && { color: theme.errorTextMenu }),
+  });
+
+  const buttonProps = {
+    style: {
+      ...styles.mobileMenuItem,
+      color: 'currentColor',
+      height: styles.mobileMinHeight,
+    },
+    activeStyle: {
+      color: 'currentColor',
+    },
+    hoveredStyle: {
+      color: 'currentColor',
+    },
+  };
+
+  const allTransactionsAreLinked = useMemo(() => {
+    return transactions
+      .filter(t => selectedTransactions.includes(t.id))
+      .every(t => t.schedule);
+  }, [transactions, selectedTransactions]);
+
+  const isMoreThanOne = selectedTransactions.length > 1;
+
+  return (
+    <View
+      style={{
+        backgroundColor: theme.floatingActionBarBackground,
+        color: theme.floatingActionBarText,
+        position: 'fixed',
+        bottom: 10,
+        margin: '0 10px',
+        width: '95vw',
+        height: 60,
+        zIndex: 100,
+        borderRadius: 8,
+        border: `1px solid ${theme.floatingActionBarBorder}`,
+        ...style,
+      }}
+    >
+      <View
+        style={{
+          flex: 1,
+          padding: 8,
+          flexDirection: 'row',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+        }}
+      >
+        <View
+          style={{
+            flexDirection: 'row',
+            alignItems: 'center',
+            justifyContent: 'flex-start',
+          }}
+        >
+          <Button
+            type="bare"
+            {...buttonProps}
+            style={{ ...buttonProps.style, marginRight: 4 }}
+            onClick={() => onClearSelectedTransactions?.()}
+          >
+            <SvgDelete width={10} height={10} />
+          </Button>
+          <Text style={styles.mediumText}>
+            {selectedTransactions.length}{' '}
+            {isMoreThanOne ? 'transactions' : 'transaction'} selected
+          </Text>
+        </View>
+        <View
+          style={{
+            flexDirection: 'row',
+            alignItems: 'center',
+            justifyContent: 'flex-end',
+            gap: 4,
+          }}
+        >
+          <Button
+            type="bare"
+            ref={editMenuTriggerRef}
+            aria-label="Edit fields"
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏in this case the aria-label makes total sense :)